config/manager/manager.yaml

```diff
@@ -69,18 +69,12 @@ spec:
       - name: manager
         image: controller:latest
         env:
-        # Note that we have a kustomization replacement that copies the above `image` into this
-        # environment variable.
+          # Note that we have a kustomization replacement that copies the above `image` into this
+          # environment variable.  It is used as a default value for the --image command line argument
+          # since it and all other arguments are implemented as CLI arguments.
         - name: IMAGE
           value: controller:latest
```

Comment:
I see IMAGE has the value 'controller:latest' by default: is this customized somehow to end up pointing to an image to quay.io ? Just wondering if the current image '"quay.io/openshift-kni/oran-o2ims-operator:latest" passed as an argument prior to these changes is still valid for you after this.

Reply:
Update : I believe you will get the new image from the CSV : https://github.com/openshift-kni/oran-o2ims/blob/main/bundle/manifests/oran-o2ims.clusterserviceversion.yaml#L1045 according the the kustomization.yaml comment, but just to double check.

Reply:
Yes, if the IMAGE variable is set then it will default to that; otherwise it will fail to launch the operator.

Reply:
When the bundle manifests are generated, the controller:latest gets replaced by the images.controller value in config/manager/kustomization.yaml, which is set to the IMG variable in the Makefile with:
```
cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
```
This is then used to generate the manifests for deploying the operator, or building the bundle.